libs/coin-modules/coin-aptos/src/network/client.ts

```diff
@@ -236,4 +236,27 @@ export class AptosAPI {
       time: new Date(Number(block.block_timestamp)),
     };
   }
+
+  async getBalances(address: string): Promise<AptosBalance[]> {
+    if (!address) {
+      return [];
+    }
+
+    const response = await this.aptosClient.getCurrentFungibleAssetBalances({
+      options: {
+        offset: 0,
+        limit: 1000,
+        where: {
+          asset_type: { _eq: APTOS_ASSET_ID },
```

Comment:
as per my question above, i think we should not have the condition for only getting native asset balances

```
function getBalance(address: string): Promise<bigint>
Returns all assets linked to the user (native, tokens, …)
```